7/GrainUnitTests/SampleDependencyUnitTest.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using Grains;
+using Microsoft.FeatureManagement;
+using Moq;
+using Orleans;
+using Orleans.TestKit;
+using Xunit;
+
+namespace GrainUnitTests
+{
+    //Unlike the standard orleans testing libraries, this allows you to run a simple unit test with dependencies.
+    // This means full silos don't have to be created which results in faster and easier to implement tests
+    public class SampleDependencyUnitTest:TestKitBase
+    {
+        [Fact]
+        public async Task InjectedDependencyCanBeVerified()
```

Comment:
This unit test does not pass when I run it locally.

![Screen Shot 2021-01-06 at 3 02 53 PM](https://user-images.githubusercontent.com/6170837/103783222-59317d80-5030-11eb-8184-9e9d6d8b2828.png)